### PR TITLE
Added option to bypass password confirmation prompt

### PR DESCRIPTION
**Pull request by AltamashShaikh**
### Description:

Added option to bypass password confirmation prompt
Fixes: #PG-1594.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * The intellectual and technical concepts contained herein are protected by trade secret or copyright law.
+ * Redistribution of this information or reproduction of this material is strictly forbidden
+ * unless prior written permission is obtained from InnoCraft Ltd.
+ *
+ * You shall use this code only in accordance with the license agreement obtained from InnoCraft Ltd.
+ *
+ * @link    https://www.innocraft.com/
+ * @license For license details see https://www.innocraft.com/license
+ */
+return array(
```
Comment (snake14) [inline config/config.php]: It's not a big deal, but it's a little odd to use `array()` here and then `[]` for the rest of the lines.

**Review by snake14 — APPROVED** [review]
Code changes look good. I confirmed that the password confirmation defaults to off when the LDAP plugin is enabled, but that the config allows it to be enabled. :+1:

File: config/config.php (hunk 1 above)
Comment (AltamashShaikh) [reply config/config.php]: @snake14 updated to `[]`

Comment (snake14) [reply config/config.php]: Thank you :+1:

**Pull request merged**